hokusai/commands/deployment.py

```diff
@@ -33,9 +33,8 @@ def promote(migration, constraint, git_remote):
   tag = ecr.find_git_sha1_image_tag(tag)
   if tag is None:
     print_red("Could not find a git SHA1 for tag %s.  Aborting." % tag)
-    return -1
+    return 1
```

Comment:
why switch to 1? this should fail the build right?

Reply:
oh is this for https://github.com/artsy/hokusai/pull/140/files#r264610865 ?

Reply:
Yeah idk why I put `-1` return code everywhere, which is essentially a `255` return code.  1 makes more sense